test/core/dialects/dialects_test.py

```diff
@@ -31,10 +32,18 @@
             y = root + ".yml"
             if y in dirlist:
                 parse_structure_examples.append((d, f, True, y))
+                has_yml = True
             # Look for the non-code included version of the structure
             y = root + "_nc.yml"
             if y in dirlist:
                 parse_structure_examples.append((d, f, False, y))
+                has_yml = True
+            if not has_yml:
+                raise (
+                    Exception(
+                        f"Missing .yml file for {os.path.join(d, f)}. See sqlfluff/test/fixtures/parser/README.md for more info."
```

Comment:
I have a Python script that generates the `.yml` file automatically. Would it be worth incorporating that somehow, in order to make it easier for contributors?

https://gist.github.com/barrywhart/868446ba70e47ea792c2a6475b6aaa47